src/pages/glossary/glossary.js

```diff
@@ -1,19 +1,25 @@
 import React from "react";
-import './glossary.scss';
+import "./glossary.scss";
 import HeroImage from "../../components/heroImage/heroImage";
 import GlossaryList from "./glossaryList";
 import Footer from "../../components/footer/footer";
+import SearchBar from "./SearchBar";
+// import glossaryTermsArray from "./glossaryhelper";
```

Comment:
The way you are importing glossaryTermsArray would only work if the glossaryhelper module exported glossaryTermsArray as a default (`export default glossaryTermsArray`). Since that is not the case, you will need to wrap the imported object in curly brackets. So line 7 should read `import { glossaryTermsArray } from "./glossaryhelper"`.